### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by schwatter**
Working script for beginners

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,41 +20,64 @@ using your smartphone.
 You are welcome to try using this as a python library for other efforts.
 A simple usage might go something like this:
 
-```python
+```import asyncio
```
Comment (edenhaus) [inline README.md:23]:
```suggestion
```python
import asyncio
```

File: README.md [hunk 2]
```diff
@@ -20,41 +20,64 @@ using your smartphone.
...
+import aiohttp
+import logging
+
+from deebotozmo.ecovacs_api import EcovacsAPI
+from deebotozmo.ecovacs_mqtt import EcovacsMqtt
+from deebotozmo.events import BatteryEvent
+from deebotozmo.vacuum_bot import VacuumBot
+from deebotozmo.ecovacs_json import EcovacsJSON
+from deebotozmo.commands import *
+
```
Comment (edenhaus) [inline README.md:33]:
```suggestion
from deebotozmo.util import md5

```

File: README.md [hunk 3]
```diff
@@ -20,41 +20,64 @@ using your smartphone.
...
+device_id 		= "3a000000000000000000000000000000"
+email			= "yourmail@deebotozmo.com"
+password_hash 	= "b6000000000000000000000000000000"
```
Comment (edenhaus) [inline README.md:36]:
```suggestion
password_hash = md5("yourPassword")
```

File: README.md [hunk 4]
```diff
@@ -20,41 +20,64 @@ using your smartphone.
...
+		api = EcovacsAPI(session, device_id, email, password_hash, continent=continent, country=country, verify_ssl=False)
+		await api.login()
 
-        devices_ = await api.get_devices()
+		devices_ = await api.get_devices()
 
-        auth = await api.get_request_auth()
-        bot = VacuumBot(session, auth, devices_[0], continent=continent, country=country, verify_ssl=False)
+		auth = await api.get_request_auth()
+		bot = VacuumBot(session, auth, devices_[0], continent="eu", country="de", verify_ssl=False)
```
Comment (edenhaus) [inline README.md:50]:
```suggestion
		bot = VacuumBot(session, auth, devices_[0], continent=continent, country=country, verify_ssl=False)
```

File: README.md [hunk 5]
```diff
@@ -20,41 +20,64 @@ using your smartphone.
...
 if __name__ == '__main__':
-    loop = asyncio.get_event_loop()
-    loop.create_task(main())
-    loop.run_forever()
+	loop = asyncio.get_event_loop()
+	loop.create_task(main())
+	loop.run_forever()
+```
+
+Save this to say cli.py and push it to /usr/local/lib/python{version}/dist-packages/deebotozmo.
```
Comment (edenhaus) [inline README.md:76]: Remove please the part of saving and running. First the file should not be under `dist-packages` as there should be only file from the package. Can create a lot of trouble if you can files there…

Second this is only an example code, someone needs to adjust it for their needs.

File: README.md [hunk 6]
```diff
@@ -20,41 +20,64 @@ using your smartphone.
...
+		async def on_battery(event: BatteryEvent):
+			# Do stuff on battery event
+			if event.value == 100:
 
-        # Battery full
+				# Battery full
 
-        # Subscribe for events (more events available)
-        bot.batteryEvents.subscribe(on_battery)
+				# Subscribe for events (more events available)
```
Comment (edenhaus) [inline README.md:61]: Wrong intention… Please use spaces as it was before and not tabs

File: README.md [hunk 7]
```diff
@@ -20,41 +20,64 @@ using your smartphone.
...
+
+from deebotozmo.ecovacs_api import EcovacsAPI
+from deebotozmo.ecovacs_mqtt import EcovacsMqtt
+from deebotozmo.events import BatteryEvent
+from deebotozmo.vacuum_bot import VacuumBot
+from deebotozmo.ecovacs_json import EcovacsJSON
+from deebotozmo.commands import *
+
+device_id 		= "3a000000000000000000000000000000"
+email			= "yourmail@deebotozmo.com"
```
Comment (edenhaus) [inline README.md:35]:
```suggestion
email = "your email or phonenumber (cn)"
```

**Review by edenhaus — CHANGES_REQUESTED** [review]
Thanks for your PR. Before merging it, please solve my comments. Specially use spaces as it was before and not tabs

File: README.md [hunk 8]
```diff
@@ -20,41 +20,64 @@ using your smartphone.
...
+		devices_ = await api.get_devices()
 
-        auth = await api.get_request_auth()
-        bot = VacuumBot(session, auth, devices_[0], continent=continent, country=country, verify_ssl=False)
+		auth = await api.get_request_auth()
+		bot = VacuumBot(session, auth, devices_[0], continent="eu", country="de", verify_ssl=False)
 
-        mqtt = EcovacsMqtt(auth, continent=continent)
-        await mqtt.subscribe(bot)
+		mqtt = EcovacsMqtt(auth, continent="eu")
```
Comment (rany2) [inline README.md:52]: `EcovacsMqtt` needs `country=` too.

Comment (edenhaus) [reply README.md:52]: Please leave `continent=continent` so it will be used the variable `continent` specified at the beginning

**Review by edenhaus — APPROVED** [review]

**Pull request merged**